data/templates/set_bmc_credentials.sh

```diff
@@ -1,9 +1,23 @@
 #!/usr/bin/env bash
 
+channel=''
+function set_channel()
+{
+    for i in {1..15}; do
+        ipmitool -I lanplus -H 172.31.128.2 -U admin -P admin user list $i &>/dev/null
+        status=$?
+        if [ "$status" -eq "0" ] ; then
+            channel=$i
+            break
```

Comment:
Perhaps there should be a check after line 15 (`set_channel`) that errors out of the script if channel is still not set?